.github/workflows/transifex_push.yml

```
@@ -0,0 +1,53 @@
+name: "Transifex Push"
+
+on:
+  workflow_call:
+    secrets:
+      tx_token:
+        required: true
+    inputs:
+      branch:
+        required: false
+        default: 'main'
+        type: 'string'
+
+jobs:
+  ci:
+    name: "Translations"
+    runs-on: ubuntu-latest
```

This will contain all the dependencies required for the `extract-locales` and `plugin-release` commands.
```suggestion
    runs-on: ubuntu-latest
    container:
      image: "ghcr.io/glpi-project/plugin-builder"
```